### DISCUSSION TITLE
How do I unsubscribe and stop recieving notifications from a toxic Gist?

### DISCUSSION THREAD

**Discussion by Espionage724** [community/community#124848] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am 100% not subscribed to it, and yet it keeps showing up on my notifications. I demand to know how to stop the notifications before I stop entertaining this platform. **I want nothing to do with or to even keep being notified of that Gist's existence.**

**Comment by CopperEagle** [comment, marked as answer]
Hey @Espionage724 

There are three buttons in the top right corner when viewing the gist. The first of the three is the subscribe/unsubscribe button (the others are star, fork). Click the first.

**Reply by CopperEagle** [reply]
Alternatively, when you get a notification, hover above it. To the right, there are again three buttons. The middle button is a bell which you can click to unsubscribe.

**Reply by Espionage724** [reply]
I was unsubscribed from it and it only showed a "subscribe" button, but I'm also not seeing notifications from it any more so I guess I'm fine with that :p Thanks!
